cluster/src/main/java/io/atomix/cluster/protocol/PhiMembershipProtocol.java

```diff
@@ -287,8 +297,13 @@ private void updateMember(GossipMember remoteMember) {
       }
       localMember.properties().putAll(remoteMember.properties());
       post(new GroupMembershipEvent(GroupMembershipEvent.Type.METADATA_CHANGED, localMember));
-    } else if (!localMember.isReachable()) {
+    } else if (!localMember.isReachable() && direct) {
+      localMember.setReachable(true);
```

Comment:
should we be checking remoteMember reachable here? is it possible for a member to receive two not reachable updates?